luatex.def

```diff
@@ -651,7 +651,7 @@
 }
 % Load package epstopdf if
 % a) LaTeX is running (plain TeX users are on their own),
-% b) and shell escape is enabled (possibly restricted),
+% XXXb) and shell escape is enabled (possibly restricted),
```

Comment:
??? seems an odd way to say it is gone (also further below)

Reply:
@FrankMittelbach yes I didn't want to re-number the list by hand while experimenting, but I'll clean up the comments before merge